### PR TITLE
[ADD] meta files

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
```
Comment (lmignon) [inline .travis.yml]: The PR is for 6.1 no?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
```
Comment (lmignon) [inline .travis.yml]: you can remove the ${VERSION} when calling travis_install_nightly and travis_run_tests

File: .travis.yml (hunk 1 above)
Comment (eLBati) [inline .travis.yml]: > The PR is for 6.1 no?

oops, right, fixed

**Comment by bwrsandman** [comment]
:+1:

File: .travis.yml (hunk 2 above)
Comment (eLBati) [inline .travis.yml]: > you can remove the ${VERSION} when calling travis_install_nightly and 
> travis_run_tests

Fixed. Thanks

**Comment by lmignon** [comment]
:+1:  LGTM only PEP8 issues

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**